### DISCUSSION TITLE
definePageMeta nested

### DISCUSSION THREAD

**Discussion by Fenish** [nuxt/nuxt#24115] [Questions] [answered]
Hello, i couldn't manage nested routes with definePageMeta. I know i can use pages for nested routing but i want to learn if it is possible. The reason i want it my route names won't be same as my .vue file names. Just asked if its possible without creating new router options 

Also i cannot set middleware via definePageMeta when i use custom router options inside router.options.ts

**Comment by danielroe** [comment]
It is indeed possible, and you should be able to set the route name that way...

If that (or the inline middleware) isn't working, perhaps you can raise an issue with a reproduction?

**Reply by Fenish** [reply, marked as answer]
I think i found the solution/problem
If your file system be like this you cannot nested route with definePageMeta
```
/pages
  |
  |---- general/     <- Not Capitalized
  |
  | General.vue  <- Capitalized
```

I fixed with lowering first key and problem solved.
